cmd/fpga_admissionwebhook/fpga_admissionwebhook.go

```diff
@@ -62,7 +62,7 @@ func addToScheme(scheme *runtime.Scheme) {
 func getTLSConfig(certFile string, keyFile string) *tls.Config {
 	sCert, err := tls.LoadX509KeyPair(certFile, keyFile)
 	if err != nil {
-		fatal(err)
+		klog.Fatalf("+%v", err)
```

Comment:
The correct specifier is `%+v`, but... my bad, this can be just `klog.Fatal(err)` indeed as `tls.LoadX509KeyPair()` is a foreign function anyway.

Reply:
np. I did wonder. sorry about the fat finger typo.... changed to plain `Fatal(err)` now.